src/connectStyle.js

```diff
@@ -106,6 +107,7 @@ export default (componentStyleName, componentStyle = {}, mapPropsToStyleNames, o
         const resolvedStyle = this.resolveStyle(context, props, styleNames);
         this.setWrappedInstance = this.setWrappedInstance.bind(this);
         this.transformProps = this.transformProps.bind(this);
+        this.isRefDefined = this.isRefDefined.bind(this);
```

Comment:
You do not need to bind this it is always called from `resolveAddedProps` and that method always has this.